public/angular/index.html

```diff
@@ -16,6 +16,24 @@
     <link rel="stylesheet" href="/css/loading-bar.min.css"/>
     <link rel="stylesheet" href="/css/tagify.css"/>
     <link rel="stylesheet" href="/css/spectrum.min.css"/>
+
+    <!-- Matomo -->
+    <script type="text/javascript">
+      var _paq = window._paq = window._paq || [];
+      /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+      _paq.push(["setCookieDomain", "*.aggiebeta.cartercenter.org"]);
+      _paq.push(['trackPageView']);
+      _paq.push(['enableLinkTracking']);
+      (function() {
+        var u="https://cartercenter.matomo.cloud/";
```

Comment:
Can we make `cartercenter` a custom setting in `secrets.json`? We probably don't want to mix all the different servers together in one Matomo dashboard.

Reply:
Also curious what happens if someone does NOT have a matomo dashboard to use. We should make sure that Aggie doesn't break if someone puts null for this value.

Reply:
> Can we make `cartercenter` a custom setting in `secrets.json`? We probably don't want to mix all the different servers together in one Matomo dashboard.

I've added a Matamo Settings to the settings.json.example. It has a boolean (send_data) to decide whether data should be collected. There's also dashboard_url, and site_id. I've also added properties for login credentials (not needed for fetching, but to access the dashboard).

Reply:
> Also curious what happens if someone does NOT have a matomo dashboard to use. We should make sure that Aggie doesn't break if someone puts null for this value.

I tried both null and empty values, and none of that broke aggie (though data collection stopped as expceted). With the additional conditional, I hope this situation would not come up in the first place, but it doesn't look like it would break aggie.

Reply:
This works great -- just FYI I removed the user/pass from secrets.json for clarity since they don't do anything. We should prefer sharing credentials in a different way if they're not needed by Aggie (otherwise users will probably think they're mandatory).